{"uuid": "e6cd6104-c381-4da2-a05e-540e396f3cde", "children": ["76cdb37f-f57d-4702-92b4-33d4a1d475b5"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "AttributeError: 'Session' object has no attribute 'testfaied'\n", "trace": "  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\PycharmProjects\\selenium_project_14.09.2023\\page_object_pattern\\tests\\conftest.py\", line 15, in setup\n    before_failed = request.session.testfaied\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1694947200423, "stop": 1694947202675}], "start": 1694947200423, "stop": 1694947202689}